modules/core/src/config.ts

```diff
@@ -290,17 +51,10 @@ export const tokens: Tokens = {
       ]
     }
   },
-  // network name for test environments (testnet tokens must be added here)
+  // network name for test environments
   testnet: {
     eth: {
-      tokens: [
-        { type: 'terc', coin: 'teth', network: 'Kovan', tokenContractAddress: '0x945ac907cf021a6bcd07852bb3b8c087051706a9', decimalPlaces: 0, name: 'ERC Test Token' },
-        { type: 'test', coin: 'teth', network: 'Kovan', tokenContractAddress: '0x1fb879581f31687b905653d4bbcbe3af507bed37', decimalPlaces: 18, name: 'Test Mintable ERC20 Token' },
-        { type: 'tbst', coin: 'teth', network: 'Kovan', tokenContractAddress: '0xe5cdf77835ca2095881dd0803a77e844c87483cd', decimalPlaces: 0, name: 'Test BitGo Shield Token' },
-        { type: 'schz', coin: 'teth', network: 'Kovan', tokenContractAddress: '0x050e25a2630b2aee94546589fd39785254de112c', decimalPlaces: 18, name: 'SchnauzerCoin' },
-        { type: 'tcat', coin: 'teth', network: 'Kovan', tokenContractAddress: '0x63137319f3a14a985eb31547370e0e3bd39b03b8', decimalPlaces: 18, name: 'Test CAT-20 Token' },
-        { type: 'tfmf', coin: 'teth', network: 'Kovan', tokenContractAddress: '0xd8463d2f8c5b3be9de95c63b73a0ae4c79423452', decimalPlaces: 18, name: 'Test Formosa Financial Token' }
-      ]
+      tokens: formattedErc20Tokens.filter(token => token.network === 'Testnet'),
     },
     ofc: {
       tokens: [
```

Comment:
Not sure what to do about OFC tokens because there seem to be more OFC tokens in statics than in here

Reply:
We should use the ofc tokens defined in statics, those should have also been added to bitgojs

Reply:
looks like there are multiple discrepancies between the OFC tokens here and the ones defined in statics. we should probably discuss the differences.